### ISSUE TITLE
[Edit] Loss Functions - neural network broken link

### ISSUE THREAD

**Issue by danitellini** [Codecademy/docs#5735]
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting

### Description (optional)

`docs/content/pytorch/concepts/nn/terms/loss-functions/loss-functions.md`

The link for neural network isn't formatted correctly in the introduction. The pointer changes, but there isn't a link attached.

`[neural network's](<(https://www.codecademy.com/resources/docs/pytorch/nn)>)`
should be
`[neural network's](https://www.codecademy.com/resources/docs/pytorch/nn)`

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

**Comment by Maverick073** [comment]
Hey @danitellini , Can you please assign this issue to me and I will start working on this .

**Comment by codecademy-docs** [comment]
Hey @Maverick073  You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.
